prac_05/state_names.py

```diff
@@ -0,0 +1,20 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
+for state in STATE_NAMES:
+    print("{:3} is {}".format(state, STATE_NAMES[state]))
+
+state = input("Enter short state: ").upper()
+while state != "":
+    if state in STATE_NAMES:
+        print(state, "is", STATE_NAMES[state])
```

Comment:
could use the .format here if you want dont have to, just if you you know want to.